oligo_designer_toolsuite/oligo_transcript_generation/_gene_transcript.py

```diff
@@ -23,17 +24,19 @@ def __init__(self, file_sequence, file_annotation):
             self.file_annotation
         )  # dataframe with annotation file
 
-    def generate_for_reference(self, block_size, dir_output="./output/annotation"):
+    def generate_for_reference(
+        self, block_size, file_gene_transcript_fasta, dir_out="output/annotation"
+    ):
         """Creates a fasta file containing the whole transcriptome and its annotation file for the reference DB. The file contains also the exon junctions, which are the union of two consecuteive exons and
```

Comment:
...union of two **consecutive** exons...
        :param block_size: size of the exon **junctions**